travis-analyzer/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, lib }:
+stdenv.mkDerivation rec {
+    pname = "travis-analyzer";
+    version = "03Jun2020";
+
+    src = fetchTarball  {
```

Comment:
Btw.: `fetchTarball` is a builtin in function and should not be used in derivations since these will fail in `strict mode` (i.e. Hydra fails to evaluate the expression). `fetchTarball` is not strict since it would work even without the hash.
fixed in f27e9d60864368e8ad5d8d9f48c640a0683aa9af